implementations/rust/ockam/ockam_command/src/node/create.rs

```diff
@@ -144,6 +144,23 @@ impl CreateCommand {
             ..cmd
         })
     }
+
+    fn parse_node_name(&self) -> crate::Result<Self> {
+        let cmd = self.clone();
+
+        if self.node_name.contains('/') {
+            if self.node_name.starts_with("/node/") {
```

Comment:
This is a little hack we should avoid as it doesn't express what we are trying to parse here. You should parse the string into a MultiAddr instance and check the protocol like it's done inside `extract_address_value`. It's more verbose, but it doesn't depend on manual/hard-coded parsing rules.

Reply:
Modified it in the latest commit to work similarly to `extract_address_value`